app/api/probe_dock/metrics_api.rb

```diff
@@ -17,6 +17,9 @@
 # along with ProbeDock.  If not, see <http://www.gnu.org/licenses/>.
 module ProbeDock
   class MetricsApi < Grape::API
+    DEFAULT_NB_DAYS_FOR_REPORTS = 30
+    DEFAULT_MAX_NB_DAYS_FOR_REPORTS = 120
```

Comment:
I would remove `DEFAULT_` from the second variable's name. The first one is the default, the second one is the max, not the "default max", whatever that means.